src/pipeline_manager.py

```diff
@@ -146,7 +168,7 @@ def predict(pipeline_name, dev_mode, submit_predictions, chunk_size, logger, par
         meta_test = meta_test.sample(2, random_state=seed)
 
     pipeline = PIPELINES[pipeline_name]['inference'](SOLUTION_CONFIG)
-    prediction = generate_prediction(meta_test, pipeline, logger, CATEGORY_IDS, chunk_size)
+    prediction = generate_prediction(meta_test, pipeline, logger, CATEGORY_IDS, chunk_size, params.num_threads)
```

Comment:
@apyskir I think we have num_threads somewhare and n_threads in other places. We should choose one I think